acceptance-tests/tests/src/test/java/org/hyperledger/besu/tests/web3j/privacy/EnclaveErrorAcceptanceTest.java

```diff
@@ -174,7 +179,17 @@ public void transactionFailsIfPartyIsOffline() {
   public void createPrivacyGroupReturnsCorrectError() {
     final Throwable throwable =
         catchThrowable(() -> alice.execute(privacyTransactions.createPrivacyGroup(null, null)));
+    final String orionMessage = JsonRpcError.CREATE_GROUP_INCLUDE_SELF.getMessage();
+    final String tesseraMessage = JsonRpcError.TESSERA_CREATE_GROUP_INCLUDE_SELF.getMessage();
+
+    assertThat(throwable.getMessage())
+        .has(matchOrionOrTesseraMessage(orionMessage, tesseraMessage));
+  }
 
-    assertThat(throwable).hasMessageContaining(JsonRpcError.CREATE_GROUP_INCLUDE_SELF.getMessage());
+  private Condition<String> matchOrionOrTesseraMessage(
+      final String orionMessage, final String tesseraMessage) {
+    return new Condition<>(
+        message -> message.contains(orionMessage) || message.contains(tesseraMessage),
+        "Message did not match either Orion or Tessera expected output");
```

Comment:
Would be useful to log the actual message

Reply:
It'd be nice, but I haven't got access to it in this pattern.